app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -372,12 +372,19 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService,
 
   override def findTestForNotification(notificationType: NotificationTestType): Future[Option[TestResultNotification]] = {
     val query = Try{ notificationType match {
-      case s: SuccessTestType => {
+      case s: SuccessTestType if s.applicationRoutes.isEmpty => {
```

Comment:
Do we need to add a check here for applicationRoute -> Fastastream for candidates who applied after we implemented the application route?